### PR TITLE
Update the opam file.

### PR DESCRIPTION
**Pull request by vbmithr**

### CODE REVIEW

File: opam [hunk 1]
```diff
@@ -1,4 +1,6 @@
-opam-version: "1"
+opam-version: "1.2"
+name: "sodium"
+version: "dev"
```
Comment (whitequark) [inline opam]: can you please not? "dev" versions are annoying to use locally, it's much better to put the next-to-be-released version into opam.

Comment (vbmithr) [inline opam]: On 01/01/2015 19:16, Peter Zotov wrote:
> can you please not? "dev" versions are annoying to use locally, it's much better
> to put the next-to-be-released version into opam.
> 

Ok, will do!

File: Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@ CURRENT_DIR := $(dir $(MAKEFILE_PATH))
 
 include $(shell ocamlc -where)/Makefile.config
 
-LIB_DIR=$(shell ocamlfind query ctypes)/..
+LIB_DIR=$(shell ocamlfind query ctypes)
```
Comment (whitequark) [inline Makefile]: You don't actually need this with `master` ctypes, `-package` adds it already.

**Comment by vbmithr** [comment]
@whitequark: I don't manage to compile it without LIB_DIR.
@dsheets: This should be mergeable now.

**Pull request merged**